source/Grid/Grid.js

```diff
@@ -687,10 +692,18 @@ export default class Grid extends Component {
     }
   }
 
+  _wrapPropertyGetter (value) {
+    return value instanceof Function
+      ? value
+      : () => value
+  }
+
   _wrapSizeGetter (size) {
-    return typeof size === 'function'
-      ? size
-      : () => size
+    return this._wrapPropertyGetter(size)
+  }
+
+  _wrapCellClassNameGetter (className) {
+    return this._wrapPropertyGetter(className)
```

Comment:
Couple of minor nits:
* Alpha-sort private methods and properties. (This just helps keep things tidy and easier to read/maintain.)
* Add a `cellStyle` property. (I try to mirror class-name and style object properties b'c there are a reasonable number of people who prefer inline styles over css classes.)

I like the generic `_wrapPropertyGetter` change. 👍